packages/portalnetwork/src/wire/utp/PortalNetworkUtp/ContentRequest.ts

```diff
@@ -19,12 +19,14 @@ export class HistoryNetworkContentRequest {
   writer?: ContentWriter
 
   constructor(
+    protocolId: ProtocolId,
     requestCode: RequestCode,
     contentKey: Uint8Array[],
     socket: UtpSocket[],
     socketKey: string,
     content: Uint8Array[] | undefined[]
   ) {
+    this.protocolId = protocolId
     this.sockets = socket
     //@ts-ignore
     this.contentKeys = contentKey
```

Comment:
Is it possible to better clarify the `contentKey` vs `contentKeys` parameters that are passed into this constructor versus the properties that are being initialized?  I'm tracking why we have seeming conflicting naming patterns between them.

Reply:
Nm, this change was easier than I thought.